shiny/ui/_input_check_radio.py

```diff
@@ -124,19 +124,26 @@ def input_switch(
     ~shiny.ui.input_radio_buttons
     """
 
-    return _input_checkbox(id, label, "form-check form-switch", value, width=width)
+    return _input_checkbox(
+        id,
+        label,
+        "bslib-input-switch form-switch",
+        value,
+        width=width,
+    )
 
 
 def _input_checkbox(
     id: str,
     label: TagChild,
-    class_: str = "form-check",
+    class_: str = "bslib-input-checkbox",
```

Comment:
Should this class be on `input_checkbox`?

Also, why does this function exist? If it's simple to do, maybe `input_checkbox` should use this one. Or maybe we should rename this.